src/pages/Integrations/Create/CustomComponents/CreatedStep.tsx

```diff
@@ -62,19 +62,10 @@ export const CreatedStep: React.FunctionComponent<ProgressProps> = (props) => {
             <Button
               variant="link"
               component="a"
-              href={`${
-                isBeta() ? '/preview' : ''
-              }/${getBundle()}/notifications/configure-events?bundle=${
+              href={`/${getBundle()}/notifications/configure-events?bundle=${
                 props.data.bundle_name
               }&tab=behaviorGroups`}
               size="lg"
-              onClick={(e) => {
-                props.onClose();
-                e.preventDefault();
-                navigate(
-                  `/${getBundle()}/notifications/configure-events?tab=behaviorGroups`
-                );
-              }}
```

Comment:
We might want to keep the navigate active. So we don't perform full page refresh.

Reply:
Since this appears after its created, I thought we would want a refreshed page to be sure we have the updated list of behavior groups?

Reply:
No need to trigger full page refresh, the page should pull the data when user lands on it via `navigate`